test/node-server/README.md

```diff
@@ -4,9 +4,9 @@ This directory contains two files:
 2. `Dockerfile.containerd`: used to run containerd
 
 ## Why?
-This is to test `cmd/plugin/node_server_test.go` and `pkg/remoteimage/pull_test.go`
+This is to run warm metal code in a container where containerd is running
 
-## How to run the tests?
+## How to use?
 Build the image
 ```shell
 $ docker build . -f Dockerfile.containerd  -t <image-name>:<tag>
```

Comment:
Instead of a separate `docker build` step, I'd suggest modifying the `docker-compose.yaml`. This way, you don't need a manual modification of the tag in the `docker-compose.yaml`

```yaml
services:
    containerd-workspace:
        image: csi-driver-image-test/containerd
        build:
          context: .
          dockerfile: Dockerfile.containerd
        privileged: true # to enable mount/unmount operations
        command: containerd
        # mount code in the current repo to `/code` path in the container
        volumes:
          - ../../../.:/code
```

The instructions then change to
```
docker compose up
```

Also two other suggestions
1. Modify the instructions to use `docker compose` as opposed to `docker-compose`. See docs - [docker compose vs docker-compose](https://docs.docker.com/compose/migrate/#docker-compose-vs-docker-compose).
2. Consider naming the image to be indicative of it being a test or csi-driver-image repo related image. 

---
P.S.
FYI, docker compose will first try [pulling then building](https://docs.docker.com/compose/compose-file/build/#consistency-with-image) the image

Reply:
Created a separate issue to address this: https://github.com/warm-metal/container-image-csi-driver/issues/133